semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -823,7 +823,7 @@ languages map onto these properties.
   <tr>
     <td><code>0x8000</code></td>
     <td><code>DEFAULT</code></td>
-    <td>Is a default parameter?</td>
+    <td>Is a default parameter or a default method?</td>
```

Comment:
We could separate these flags, especially since there may be confusion between default getters and default methods.

Do you have any suggestions how to call the flags? `DEFAULT_PARAM` and `DEFAULT_METHOD` unfortunately don't work due to the reason mentioned above.